RecoHI/HiJetAlgos/plugins/HiFJRhoFlowModulationProducer.h

```diff
@@ -0,0 +1,34 @@
+#ifndef RecoHI_HiJetAlgos_HiFJRhoFlowModulationProducer_h
+#define RecoHI_HiJetAlgos_HiFJRhoFlowModulationProducer_h
```

Comment:
here and in other cases of .h files declaring EDProducers, please merge the .h file content into .cc; there is no need for a header file